src/home/apis/postChangePassword.ts

```diff
@@ -0,0 +1,29 @@
+import { AxiosError } from 'axios';
+
+import { authClient } from '@/apis';
+import { SessionTokenType } from '@/home/types/password.type';
+
+import { AuthErrorData, PostAuthSignInData } from '../types/Auth.type';
+
+interface changePasswordProps {
+  email: string;
+  newPassword: string;
+  sessionToken: SessionTokenType;
+}
+
+export const postChangePassword = async (
+  props: changePasswordProps
+): Promise<PostAuthSignInData> => {
+  const { email, sessionToken, newPassword } = props;
+
+  try {
+    const res = await authClient.post('/auth/change-password', {
+      email: email,
+      newPassword: newPassword,
+      sessionToken: sessionToken.sessionToken,
+    });
+    return res.data;
+  } catch (error) {
+    throw error as AxiosError<AuthErrorData>;
+  }
+};
```

Comment:
`axios`는 실패한 HTTP 요청에 대해 자동으로 에러를 던져주기 때문에 try catch로 에러를 잡아서 던져줄 필요는 없을 것 같아요..!

https://github.com/axios/axios?tab=readme-ov-file#handling-errors

```suggestion
export const postChangePassword = async ({
  email,
  sessionToken,
  newPassword,
}: changePasswordProps): Promise<PostAuthSignInData> => {
  const res = await authClient.post('/auth/change-password', {
    email: email,
    newPassword: newPassword,
    sessionToken: sessionToken.sessionToken,
  });

  return res.data;
};
```

Reply:
[4f1f578](https://github.com/yourssu/Soomsil-Web/pull/196/commits/4f1f578460194c55280cac29a1f0db8c6029e224)
반영 했습니다 감사합니다!